src/views/Patient/MedReqDropDown/MedReqDropDown.tsx

```diff
@@ -170,7 +170,10 @@ function MedReqDropDown({
         },
         [resourceId]
       );
-      const cdsUrl = getMedicationSpecificRemsAdminUrl(request, hook.hookType);
+      const cdsUrl = env.get('USE_INTERMEDIARY').asBool()
```

Comment:
Maybe just move this entirely to a function in the utils? Then you might not need to export the getIntermediaryRemsAdminUrl function and would keep the env reading all in the utils.